tasks/sep/006-1-sc-changes/VALIDATION.md

```diff
@@ -37,34 +63,29 @@ Links:
 **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
 **After:** `0x0000000000000000000000004416c7fe250ee49b5a3133146a0bbb8ec0c6a321` <br/>
-**Meaning:** The `LivenessGuard` address is set to [0x4416c7fe250ee49b5a3133146a0bbb8ec0c6a321](https://sepolia.etherscan.io/address/0x4416c7fe250ee49b5a3133146a0bbb8ec0c6a321). The key can be validated by the key in the [Guard Manager](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/base/GuardManager.sol#L30).
-
-The following three changes are all updates to the `modules` mapping, which is in [slot 1](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L10).
+**Meaning:** The `LivenessGuard` address is set to [0xc26977310bC89DAee5823C2e2a73195E85382cC7](https://sepolia.etherscan.io/address/0xc26977310bC89DAee5823C2e2a73195E85382cC7). The key can be validated by the key in the [Guard Manager](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/base/GuardManager.sol#L30).
 
-**Key:** `0xacbf9592ae4032ad5c93b5c8322c5e0fca1cc9ae67f551dee5a8b6f78041eabf` <br/>
-**Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
-**After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
-**Meaning:** The `LivenessModule` at `0x812b1fa86be61a787705c49fc0fb05ef50c8fedf` is now pointing to the sentinel module at `0x01`.
-  This is `modules[0x812b1fa86be61a787705c49fc0fb05ef50c8fedf]`, so the key can be
-    derived from `cast index address 0x812b1fa86be61a787705c49fc0fb05ef50c8fedf 1`.
+The following two changes are both updates to the `modules` mapping, which is in [slot 1](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/examples/libraries/GnosisSafeStorage.sol#L10).
 
 **Key:** `0xcc69885fda6bcc1a4ace058b4a62bf5e179ea78fd58a1ccd71c22cc9b688792f` <br/>
 **Before:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
-**After:** `0x000000000000000000000000ed12261735ad411a40ea092ff4701a962d25ca21` <br/>
-**Meaning:** The sentinel module (`address(0x01)`) is now pointing to the `DeputyGuardianModule` at [0xed12261735aD411A40Ea092FF4701a962d25cA21](https://sepolia.etherscan.io/address/0xed12261735aD411A40Ea092FF4701a962d25cA21).
+**After:** `0x000000000000000000000000eb3ef34acf1a6c1630807495bcc07ed3e7b0177e` <br/>
+**Meaning:** The sentinel module (`address(0x01)`) is now pointing to the `LivenessModule` at [`0xEB3eF34ACF1a6C1630807495bCC07ED3e7B0177e`](https://sepolia.etherscan.io/address/0xEB3eF34ACF1a6C1630807495bCC07ED3e7B0177e).
   This is `modules[0x1]`, so the key can be
     derived from `cast index address 0x0000000000000000000000000000000000000001 1`.
 
-**Key:** `0xfa5b7c957b60505335b308d4cb1897b38d70b1c1ed76638190c5511b8707ee1b` <br/>
+**Key:** `0xd94b9fe9ff6b9f2bdd33e60c54ffff22143c4794aae3d24091fa3ef8ace26714` <br/>
```

Comment:
The key changes because of the module address change. I pulled this from the tenderly simulation, and it matches the `cast index` output

```suggestion
**Key:** `0x5ebcfc77c53cd2a46ad21b72f1b6a878f7bea47b3baad3c0383053a9361006b8` <br/>
```

Reply:
Ah, thank you